vagrant_env/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+set -u
+#set -x
+
+VAGRANT_HOST="rockstor-installer"
+
+vagrant up ${VAGRANT_HOST}
+
+VAGRANT_PATH="/home/vagrant"
+#vagrant ssh -c "ls -l ${VAGRANT_PATH}" ${VAGRANT_HOST}
+
+CODE_PATH="${VAGRANT_PATH}"
+echo "$(basename ${BASH_SOURCE[0]}): CODE_PATH is '${CODE_PATH}'"
+
+vagrant ssh -c "cd ${CODE_PATH}; /vagrant/run_kiwi.sh" ${VAGRANT_HOST}
```

Comment:
Again this should have the permissions set correctly in the repo and on pulling a fresh clone I do see +x on the two bash scripts.

Reply:
@phillxnet, regarding KVM there is a vagrant plugin to provide a KVM provider. This can be added to the plugin list at the top of the Vagrant file.

The Vagrantfile can then have multiple providers. By default it chooses the first provide it finds in the Vagrant, but an option  '--provider=<name of provider>' can be used to force the provider of your choice.

eg.

vagrant up --provider=virtualbox
or 
vagrant up --provider=kvm

I don't have kvm to test it with at present...

Reply:
FYI the /vagrant directory in the VM is a mounting of the 'vagrant_env'. So in theory the run_kiwi.sh should have the same permissions as it's checked out with and this change should be necessary.

Reply:
Same mistake as previously on my part I'm afraid. Thanks again. And thanks for the added info re mounting info.

Reply:
Phew! At least I wasn't going mad this time!

Reply:
Do you 'the review' resolve this thread when you do a final review?

Reply:
Yes, or as I go along of course. Resolving this bit now so it's put to bed.